### DISCUSSION TITLE
Printer blinking but no printing start

### DISCUSSION THREAD

**Discussion by SankyuTech** [php-aidc/label-printer#15] [Q&A]
Hi, im trying to integrate my project with Honeywell thermal printer PC42t Plus. The printer blinking when i send print command to the printer, but the printer not start to print. Here is the code that i use.

```
$label = Label::create(Unit::MM(), 75, 35)
        ->charset(Charset::UTF8())
        ->add(Element::textBlock(20, 20, 'Hello!', 'Univers', 20));

(new Printer(new NetworkConnector(env('PRINTER_IP')), CompilerFactory::tspl()))->print($label);
```

Thank You.

**Comment by xavehoo** [comment]
Try this:

(new Printer(new NetworkConnector('PRINTER_IP'), Compiler::create(new Tspl())))->print($label);

**Comment by sergio773** [comment]
Exactly same issue for me with same printer, solution suggested by @xavehoo is not working for me. 

@SankyuTech did you solve it?
